python/mpt/tests/test_mpt.py

```diff
@@ -0,0 +1,302 @@
+import json
+
+from ethereum.cancun.fork_types import Account, Address, encode_account
+from ethereum.cancun.state import get_account
+from ethereum.cancun.trie import bytes_to_nibble_list, nibble_list_to_compact
+from ethereum.crypto.hash import Hash32, keccak256
+from ethereum_rlp import rlp
+from ethereum_spec_tools.evm_tools.loaders.fixture_loader import Load
+from ethereum_types.bytes import Bytes, Bytes32
+from ethereum_types.numeric import U256, Uint
+
+from mpt import EMPTY_TRIE_ROOT_HASH, StateTries
+
+TEST_PATH = "./data/1"
+SUB_PATH = "inputs"
+FILE_NAME = "22009357.json"
+
+# A set of encoded storage keys and values for testing
+STORAGE_KEYS: list[Bytes32] = [U256(i).to_be_bytes32() for i in range(1, 6)]
+KECCAK_STORAGE_KEYS: list[Bytes32] = [Bytes32(keccak256(key)) for key in STORAGE_KEYS]
+
+STORAGE_VALUES: list[U256] = [U256(i) for i in range(1, 6)]
+RLP_STORAGE_VALUES: list[Bytes] = [rlp.encode(value) for value in STORAGE_VALUES]
+
+ADDRESSES: list[Address] = [
+    Address(bytes.fromhex(f"000000000000000000000000000000000000000{i:x}"))
+    for i in range(1, 6)
+]
+
+# Standard test account
+TEST_ACCOUNT = Account(
+    nonce=Uint(10), balance=U256(1000), code=bytes.fromhex("deadbeef")
+)
+
+
+class TestStateTries:
+    def test_from_json(self):
+        mpt = StateTries.from_json(f"{TEST_PATH}/{SUB_PATH}/{FILE_NAME}")
+        assert mpt is not None
+
+    def test_get(self):
+        mpt = StateTries.from_json(f"{TEST_PATH}/{SUB_PATH}/{FILE_NAME}")
+
+        random_address = list(mpt.access_list.keys())[0]
+
+        key = keccak256(random_address)
+
+        result = mpt.get(key)
+
+        assert result is not None
```

Comment:
is `is not None` sufficient"? Don't we want expected values to prevent eventual regressions?